lib/app.rb

```diff
@@ -44,7 +37,11 @@ class App < Sinatra::Base
   # Search for Manifests
   get '/api/:version/manifest/search' do
     content_type :json
-    Manifest.all.to_json
+    if !params[:q] && !params[:aq]
```

Comment:
I was aiming for compatibility with google.com: https://www.google.com/#q=search+query+param+name